common/src/test/java/org/apache/sedona/common/raster/RasterBandAccessorsTest.java

```diff
@@ -74,6 +74,116 @@ public void testBandNoDataValueIllegalBand() throws FactoryException, IOExceptio
         assertEquals("Provided band index 2 is not present in the raster", exception.getMessage());
     }
 
+    @Test
```

Comment:
For the tests, since we're dealing with means and std which are small, can we see if delta of 1e-6 is sufficient instead of 0.1d?